lib/recipe_cli/api.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Api
```

Comment:
[STYLE]

I'd recommend capitalizing class names exactly as you would writing them elsewhere - you'd call it an API in text, so I'd call the class `API`, too.

```suggestion
class API
```